pkg/sql/sem/asof/as_of.go

```diff
@@ -226,6 +226,10 @@ const (
 	// ReplicationCutover is when the DatumToHLC() is used for an
 	// ALTER VIRTUAL CLUSTER ... COMPLETE REPLICATION statement.
 	ReplicationCutover
+
+	// ShowTenantFingerprint is when the DatumToHLC() is used for an SHOW
+	// EXPERIMENTAL_FINGERPRINTS FROM TENANT ... WITH START TIMESTAMP statement.
+	ShowTenantFingerprint
```

Comment:
Should this have the same constraints as AsOf?  Perhaps by making it have the same value as AsOf?

Reply:
good point, i think it should. assigned it the same value as AsOf.